react/src/components/CustomizedImageList.tsx

```diff
@@ -101,17 +120,42 @@ const CustomizedImageList: React.FC<PropsWithChildren> = ({ children }) => {
     `);
 
   const columns: ColumnsType<CommittedImage> = [
+    {
+      title: t('environment.FullImagePath'),
+      key: 'fullImagePath',
+      render: (row) => (
+        <Flex gap={'xxs'}>
+          <TextHighlighter keyword={imageSearch}>
+            {getImageFullName(row) || ''}
+          </TextHighlighter>
```

Comment:
Commented on [Previous Stack](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2794/refactor-MyEnvironmentPage-into-two-components#comment-PRRC_kwDOCRTcws5szp4Z) about using the CopyButton component and the criteria for using copyable in Typography.Text.